### PR TITLE
TMT for multiple tests

### PR DESCRIPTION
**Pull request by ZelenyMartin**
Fixing various old attributes which make troubles for tmt-1.16

### CODE REVIEW

File: aide/Sanity/aide-output-to-multiple-report_url-streams/main.fmf [hunk 1]
```diff
@@ -11,12 +11,10 @@ contact: Dalibor Pospíšil <dapospis@redhat.com>
...
-- coreutils
-- rsyslog
+  - aide
+  - coreutils
+  - rsyslog
 duration: 5m
 enabled: true
 tier: '1'
-relevancy: |
-    distro = rhel-4, rhel-5: False
```
Comment (sopos) [inline aide/Sanity/aide-output-to-multiple-report_url-streams/main.fmf:21]: adjust:
  - enabled: false
    when: distro < rhel-6

File: rsyslog/Regression/bz1223566-RFE-imrelp-does-not-support-binding-to-a-specific/main.fmf [hunk 2]
```diff
@@ -6,15 +6,13 @@ contact: Dalibor Pospíšil <dapospis@redhat.com>
...
-- policycoreutils-python
-- psmisc
+  - rsyslog
+  - rsyslog-relp
+  - policycoreutils-python
+  - psmisc
 duration: 30m
 enabled: true
-relevancy: |
-    distro = rhel-4, rhel-5: False
```
Comment (sopos) [inline rsyslog/Regression/bz1223566-RFE-imrelp-does-not-support-binding-to-a-specific/main.fmf:19]: add
```
adjust:
  - enabled: false
    when: distro < rhel-6
```

**Comment by ZelenyMartin** [comment]
I don't consider `adjust` to be needed here.

**Comment by sopos** [comment]
Ok, if that's the intention..

**Pull request merged**